migrate/20240507_pci_device.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:pci_device) do
+      column :id, :uuid, primary_key: true
+      column :slot, :text, null: false
+      column :device_class, :text, null: false
+      column :vendor, :text, null: false
+      column :device, :text, null: false
+      column :numa_node, :Integer, null: true
+      column :iommu_group, :Integer, null: false
+      column :enabled, :bool, null: false, default: true
+      foreign_key :vm_host_id, :vm_host, type: :uuid, null: false
+      foreign_key :vm_id, :vm, type: :uuid, null: true
```

Comment:
We will probably not have many PCI devices, initially, but we might want to add an index at some point on `vm_id`.

Reply:
Added an index.